index.js

```diff
@@ -20,7 +20,7 @@ module.exports = function(el) {
   var candidate, candidateIndex;
   for (var i = 0, l = candidates.length; i < l; i++) {
     candidate = candidates[i];
-    candidateIndex = candidate.tabIndex;
+    candidateIndex = parseInt(candidate.getAttribute('tabindex'), 10) || candidate.tabIndex;
```

Comment:
I have found that I only need `focusable="false"` --- curious why you're seeing something different. Nothing in this PR addresses *that*, tright?